app/views/bookmarks/_filters.html.erb

```diff
@@ -85,6 +85,16 @@
       </dt>
       <dd class="more group">
         <dl>
+          <dt id="toggle_work_words" class="filter-toggle words"><%= t("bookmarks.filters.word_count") %></dt>
```

Comment:
Is it possible to use lazy i18n keys here, or is it weird because of this being a partial?

(If this is for re-use, our translators generally prefer repetition instead because it fits their process better.)

Reply:
I think I figured out after all what you meant and changed it like you wanted, but please let me know if I misunderstood!